.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are still diffs in this file relative to the base, such that it shows up in your aggregate patch. Rather than manually trying to fix it you need to revert it to the state it was in the develop branch (assuming that is what your patch is based off of). You can copy it manually or more easily doing this, while on your branch:

```
git checkout develop -- .idea/vcs.xml
```

And then commit and push it. It should then no longer show up in the aggregate patch.

Reply:
Thanks, I did a big mess (sorry). Seems to be ok now. Sorry for the long commit list.